### ISSUE TITLE
About filter_bg_thresh

### ISSUE THREAD

**Issue by cainiaoshidai** [VinAIResearch/ISBNet#1]
Hi, 
    It's excellent work. I found that when running your code, a parameter `filter_bg_thresh` will cause the output loss to be 0. Effective loss can only be produced if the threshold exceeds this parameter. I don't know if my understanding is correct. Can you tell me what this parameter does？
looking forward to your reply.
Best.

**Comment by ngoductuanlhp** [comment]
Hi @cainiaoshidai,
Thanks for your interest in our work!
`filter_bg_thresh` is a soft threshold to filter out points belonging to background classes (floor and wall). The 0 output loss only happens if you train the entire network from scratch. Please refer to our [training guideline](https://github.com/VinAIResearch/ISBNet/blob/master/docs/TRAIN.md) in which we first pre-train the backbone then train the entire model.
Best.

**Comment by cainiaoshidai** [comment]
> Hi @cainiaoshidai, Thanks for your interest in our work! `filter_bg_thresh` is a soft threshold to filter out points belonging to background classes (floor and wall). The 0 output loss only happens if you train the entire network from scratch. Please refer to our [training guideline](https://github.com/VinAIResearch/ISBNet/blob/master/docs/TRAIN.md) in which we first pre-train the backbone then train the entire model. Best.

Thanks for your reply. I made a mistake while reading the training guideline. By the way, I want to train in my own dataset. But my dataset doesn't have `RGB` feature.  So whether the `RGB` feature will have a significant influence on the result of indoor and outdoor datasets. Can you give me some advice?
Thanks.
Best.

**Comment by ngoductuanlhp** [comment]
I think the lack of RGB features is not a big problem. However, when adapting to a new dataset, you should fine-tune several hyper-parameters (as suggested in [custom_data_guide](https://github.com/thangvubk/SoftGroup/blob/main/docs/custom_dataset.md) of SoftGroup) such as the `voxel_scale`, `instance_head_cfg.radius`, `instance_head_cfg.neighbor` in the cfg file.
Best.

**Comment by cainiaoshidai** [comment]
> 

Thanks. I will have a try.
Best.
